### PR TITLE
Create shared routes

### PR DESCRIPTION
**Pull request by TylerNoblett**
allow API access to most web routes. To do this, I put all the routes from web that either 1. weren't in API (excluding routes that use session data or authscheme data) or 2. were identical to the API into a shared routes file. Note that there were a few routes that had the same path in API and web but had slightly different implementations, so I left those in their respective route groupings.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jkennedyvz** [comment]
Hi @TylerNoblett, we want to add the API functionality into web, but not the other way around. 

We'll keep the web service as our only required container, and allow users that wish to keep the API service separate to do so. This should make transitioning deployments a bit easier as well.

File: backend/server/api.go [hunk 1]
```diff
@@ -25,64 +24,17 @@ func API(r chi.Router, db *database.Connection, contentStore contentstore.Store,
...
 		bindAPIRoutes(r, db, contentStore)
 	})
 }
 
 func bindAPIRoutes(r chi.Router, db *database.Connection, contentStore contentstore.Store) {
-	route(r, "GET", "/operations", jsonHandler(func(r *http.Request) (interface{}, error) {
-		return services.ListOperations(r.Context(), db)
-	}))
-
 	route(r, "GET", "/checkconnection", jsonHandler(func(r *http.Request) (interface{}, error) {
```
Comment (jkennedyvz) [inline backend/server/api.go:34]: We'll need /checkconnection available from web.go so clients can verify their configuration.

**Comment by TylerNoblett** [comment]
> Hi @TylerNoblett, we want to add the API functionality into web, but not the other way around.
> 
> We'll keep the web service as our only required container, and allow users that wish to keep the API service separate to do so. This should make transitioning deployments a bit easier as well.

ah gotcha - I misunderstood. That makes sense!

**Comment by TylerNoblett** [comment]
Resolved in #898

**Pull request closed without merging**